src/smc-project/jupyter/package.json

```diff
@@ -14,8 +14,8 @@
   "dependencies": {
     "enchannel-zmq-backend": "^2.6.1",
```

Comment:
(I just published this too).

We're about to have this setup where these dependent projects are together in one monorepo, though still separate node modules.